src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/ColorPicker.xaml

```diff
@@ -508,7 +510,9 @@
                                 <Setter Property="Template">
                                     <Setter.Value>
                                         <ControlTemplate TargetType="Thumb">
-                                            <Border BorderBrush="{ThemeResource SliderThumbBorderBrush}"
+                                            <Border 
```

Comment:
Maybe best to add a little comment for the context that the issue is caused by https://github.com/unoplatform/uno/issues/6457 no?
And same for `Slider` in case there is no comment there to give a bit more context ;)